src/Core/Commander/Providers/TiledImageTools.js

```diff
@@ -0,0 +1,126 @@
+import * as THREE from 'three';
+import Fetcher from './Fetcher';
+import CacheRessource from './CacheRessource';
+import IoDriver_XBIL from './IoDriver_XBIL';
+import Projection from '../../Geographic/Projection';
+import CoordWMTS from '../../Geographic/CoordWMTS';
+import { UNIT } from '../../Geographic/Coordinates';
+
+const SIZE_TEXTURE_TILE = 256;
+
+// CacheRessource is necessary for neighboring PM textures
+// Info : THREE.js have cache image https://github.com/mrdoob/three.js/blob/master/src/loaders/ImageLoader.js#L25
```

Comment:
THREE.js uses cache, but never evict it, never check its validity, doesn't have any limits in size. It's really bad. I'm just commenting on the `Info` comment. No action needed in this PR :-)